app/assets/scripts/map.js

```diff
@@ -157,6 +157,40 @@ $(document).ready(function() {
     });
 });
 
+function getNextDate(startDate) {
+    date = startDate.getDate();
+    month = startDate.getMonth();
+    year = startDate.getFullYear();
+    MAX_MONTH = 11;
+    monthDays = {
+        0: 31,
+        1: 28,
+        2: 31,
+        3: 30,
+        4: 31,
+        5: 30,
+        6: 31,
+        7: 31,
+        8: 30,
+        9: 31,
+        10: 30,
+        11: 31
+    }
+    if ((month == 1) && ((year % 400 == 0) || ((year % 100 != 0) && (year % 4 == 0)))) {
```

Comment:
Wait, what exactly are we supposed to set the end date to? The current day? Or "tomorrow"? - now that I'm reading your comment, I'm not actually sure what this function is used for lol. What does `getNextDate` mean?/what is "next"?

I'd maybe consider renaming the function and definitely would add a docstring or comment here!